### PR TITLE
Make `-a`, `-alias` and `-arch` consistent on Windows and non-Windows platforms

### PR DESCRIPTION
**Pull request by dougbu**
Takes @yaakov-h's work on `dnvm.sh` (see #419) and does the same with `dnvm.ps1`
- have a look at second commit to see exact differences from #419

### CODE REVIEW

File: src/dnvm.ps1 [hunk 1]
```diff
@@ -1106,12 +1106,12 @@ function dnvm-alias {
         [Parameter(Position=1)]
         [string]$Version,
 
-        [Alias("arch")]
+        [Alias("arch", "a")]
```
Comment (dougbu) [inline src/dnvm.ps1]: Order here is not important to PowerShell but `dnvm-help` displays only the first alias.

File: src/dnvm.ps1 [hunk 2]
```diff
@@ -1174,17 +1174,16 @@ function dnvm-unalias {
...
         [ValidateSet("", "x86", "x64", "arm")]
         [Parameter(Mandatory=$false)]
         [string]$Architecture = "",
 
         [Alias("r")]
-        [ValidateSet("", "clr", "coreclr")]
+        [ValidateSet("", "clr", "coreclr", "mono")]
```
Comment (dougbu) [inline src/dnvm.ps1]: Minor issue I fixed as I was driving by.

**Comment by dougbu** [comment]
Note I haven't updated the `dnvm.ps1` tests yet.

**Comment by dougbu** [comment]
Updated to fix tests

**Comment by BrennanConroy** [comment]
LGTM :shipit:

**Comment by dougbu** [comment]
- 6f872cf -- thanx @yaakov-h 
- f0a8ba2

**Comment by muratg** [comment]
@dougbu [done](https://github.com/aspnet/Announcements/milestones/1.0.0-rc1) :smile:

**Pull request closed without merging**